HW0/P4/P4.py

```diff
@@ -51,12 +58,31 @@
     # A = ?
     # a = ?
     # s = ?
-
+    
+    A = np.array( [ [1,0,0, dt,  0,  0], 
```

Comment:
I defined A as you did, but one of my peer reviewers suggested that np.diag() could be a more efficient method, especially for larger matrices than this one.  I found it a useful comment so I thought I'd pass it on!